helmfile/values/sc-servicemonitor.yaml.gotmpl

```diff
@@ -4,6 +4,12 @@ defaultRules:
 
 rookMonitor:
   enabled: {{ .Values.monitoring.rook.enabled }}
+  relabelings:
+  - targetLabel: cluster
+    replacement: service_cluster
```

Comment:
Will not this one be in wc as well?

Reply:
Then you might need to add some wrapper around this to only affect the values in a `{{ if eq .Environment.Name "service_cluster" }}` otherwise we will get this in wc as well

Reply:
yes, for wc we do not do relabelling at the servicemonitor level
we use externalLabels in prometheusSpec instead 
https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/kube-prometheus-stack-wc.yaml.gotmpl#L56

Reply:
We might want to add `externalLabels` here as well. Just to be sure that we get these labels for all metrics

Reply:
@Xartos will add that :+1:

Reply:
there is a separate issue for that, as it may have some undesired effects

Reply:
I do not this we need an extra wrapper, we already have it here:
https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/50-applications.yaml#L200

but correct me if I am wrong

Reply:
Do not add external labels in prometheus-sc.
Previously there were some caveats when using external labels do definitely do not fit our use of prometheus in the service cluster.

Reply:
Externallabels are as the name suggest - external 😄, so if prometheus is configured with externallabels set, they will be removed from the metrics once fetched from remote storage. At least that is how i remember it.

Reply:
Right, just found an old message about it. It seems to be a prometheus with influx as backend issue, so (at least with thanos) this shouldn't be an issue anymore